### PR TITLE
Upgrade wp-prettier@2.0.5

### PR DESCRIPTION
**Pull request by scinos**
#### Changes proposed in this Pull Request

* Update to `wp-prettier@2.0.5-beta-4`. Includes fixes for https://github.com/Automattic/wp-prettier/issues/25 and https://github.com/Automattic/wp-prettier/issues/28
* Reformat files with `yarn reformat-files`

#### Testing instructions

* Checkout the branch, run `yarn reformat-files` and verify there are no extra changes.

### CODE REVIEW

File: packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js [hunk 1]
```diff
@@ -281,7 +281,7 @@ const StripeFields = styled.div`
 const CreditCardFieldsWrapper = styled.div`
 	padding: 16px;
 	position: relative;
-	display: ${( props ) => ( props.isLoaded ? 'block' : 'none') };
+	display: ${( props ) => ( props.isLoaded ? 'block' : 'none' )};
```
Comment (sirreal) [inline packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js:284]: I would've expected space in here: `${( props ) => ( props.isLoaded ? 'block' : 'none') }`

File: packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js [hunk 2]
```diff
@@ -340,7 +340,7 @@ const StripeFieldWrapper = styled.span`
 		font-size: 16px;
 		box-sizing: border-box;
 		border: 1px solid
-			${( props ) => ( props.hasError ? props.theme.colors.error : props.theme.colors.borderColor) };
+			${( props ) => ( props.hasError ? props.theme.colors.error : props.theme.colors.borderColor )};
```
Comment (sirreal) [inline packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js:343]: I would've expected space in here: `${( props ) => ( props.hasError ? props.theme.colors.error : props.theme.colors.borderColor )};
`

File: packages/composite-checkout/src/components/checkout-steps.js [hunk 3]
```diff
@@ -649,7 +649,7 @@ const StepNumber = styled.div`
 	// Reason: The IE media query needs to not have spaces within brackets otherwise ie11 doesn't read them
 	// prettier-ignore
 	@media all and (-ms-high-contrast:none), (-ms-high-contrast:active) {
-		z-index: ${ ( props ) => ( props.isComplete ? '0' : '1')  };
+		z-index: ${ ( props ) => ( props.isComplete ? '0' : '1' ) };
```
Comment (sirreal) [inline packages/composite-checkout/src/components/checkout-steps.js:652]: Tagged template with space inside the `${}` as I'd expect.

File: packages/composite-checkout/src/components/checkout-steps.js [hunk 4]
```diff
@@ -629,7 +629,7 @@ const StepNumberInnerWrapper = styled.div`
 	transform-origin: center center;
 	transition: transform 0.3s 0.1s ease-out;
 	transform-style: preserve-3d;
-	transform: ${( props ) => ( props.isComplete ? 'rotateY(180deg)' : 'rotateY(0)') };
+	transform: ${( props ) => ( props.isComplete ? 'rotateY(180deg)' : 'rotateY(0)' )};
```
Comment (sirreal) [inline packages/composite-checkout/src/components/checkout-steps.js:632]: Same file, no space inside tagged template `${}` 😕

**Review by sirreal — APPROVED** [review]
This is looking better. I do see what appears to be inconsistent behavior inside tagged templates. This may be related to some prettier-ignore comments.

Looking at the prettier playground, it does appear to format these so perhaps inserting space in the `${}` is something `wp-prettier` is missing. [The prettier playground formats these lines identically](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBlADgQzDgHkAnAOQj0LkwF5MMBPAGzgBMA6dgSwDcABgB0oQmDFL4oaAGYRSAWySYAJMAAUuUhFxoAlHQB8m7bs480AYQgLcbeAH4A5Npj54ATXUBGABwAGdjgAcz0nJBcIN091fzC9AF8AbhFMNMwJKVl5JVVgTHVMLR00TD16QwKi0zRzKxs7OHhMB0xI6LgvP0CQsMxldvdO2L7y5JEBFKgQABoQHRgedGRQfFJtAHcABTWENGQQfBYN-CZ9uYAjSTAAaybqMB4oYOQJAFc4OYALGAUWAHUvjx4GgCERsHtgfxgUwDmA0OcQE80HBSDAtpJggp8MgZEcUXMAFZoAAeACFrncYNh8Ao4AAZJ5wXH4z4gYkk7BPYJsACKbyizKQeJYBJABFIKNIBwu+AucBYs3FpCeMH+PHYMC+yACc2KKP+klwBy0cClfGZcwAjgL4BiSgd8GgALRQOAcDhK0hwG08b0Y-BYnHC1lzFEKHivUgfMPcvm2oUisVuC7qzXapAAJjmmR4LG51gU2IOZoArEq3iiACpy-Yh0VsvgfACSUCCsGwYBVuBgAEE29gYKxE6yEgkgA), as I'd expect:

```js
const SpaceOrNoSpace = styled.div`
	transform: ${(props)=>(props.isComplete?'rotateY(180deg)':'rotateY(0)')};
    transform: ${ ( props ) => ( props.isComplete ? 'rotateY(180deg)' : 'rotateY(0)' ) };
`;
```

**Comment by scinos** [comment]
Blocked by https://github.com/Automattic/wp-prettier/issues/28

**Review by sirreal — APPROVED** [review]
I did a quick scan over the changes and they look good. Shall we land this or release wp-prettier@2.0.5 first and land that?

**Comment by scinos** [comment]
Updated to `2.0.5`

**Review by sirreal — APPROVED** [review]
🚀

**Pull request merged**